app/views/projects/_delete_confirmation.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="modal-dialog">
+  <div class="modal-content">
+    <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title"><%= t('projects.delete project') %></h4>
+    </div>
+    <div class="modal-body">
+      <p> <%= t('projects.index.are you sure you want to delete this project_html', project_name: @project.name) %></p>
+      <p> <%= t('projects.delete_confirmation') %></p>
+      
+      <div class="form-group">
+        <%= form_tag(@project, method: :delete) do %>
```

Comment:
Avoid using instance vars inside partials, prefer to pass them as parameters of the partial to maximize reuse.